### ISSUE TITLE
Do not enable SSE2 by default

### ISSUE THREAD

**Issue by mgorny** [dchest/pyblake2#11]
As I've mentioned in #6, the pure SSE2 version of BLAKE2 is slower than the pure C version. I've reported this to Python upstream as [bpo31834](https://bugs.python.org/issue31834) and they've merged my patch that disables the optimized code unless at least SSSE3 is available.

Curious enough, it turns out that BLAKE2 upstream actually recommends against using pure SSE2 as they are perfectly aware that it is slow.

**Comment by dchest** [comment]
Thanks for research! On my machine BLAKE2b is indeed slightly slower with SSE2 than REGS, but BLAKE2s is much faster:

```
64 bytes                    SSE2            REGS

 pyblake2.blake2b    	    72 MB/s  	    74 MB/s 
 pyblake2.blake2s    	    95 MB/s  	    85 MB/s
                                                    
128 bytes                                           
                                                    
 pyblake2.blake2b    	   145 MB/s  	   147 MB/s
 pyblake2.blake2s    	   154 MB/s  	   132 MB/s
                                                    
1024 bytes                                          
                                                    
 pyblake2.blake2b    	   377 MB/s  	   393 MB/s
 pyblake2.blake2s    	   342 MB/s  	   247 MB/s
                                                    
2047 bytes                                          
                                                    
 pyblake2.blake2b    	   430 MB/s  	   454 MB/s
 pyblake2.blake2s    	   373 MB/s  	   264 MB/s
                                                    
2048 bytes                                          
                                                    
 pyblake2.blake2b    	   408 MB/s  	   429 MB/s
 pyblake2.blake2s    	   354 MB/s  	   257 MB/s
                                                    
1000000 bytes                                       
                                                    
 pyblake2.blake2b    	   514 MB/s  	   523 MB/s
 pyblake2.blake2s    	   410 MB/s  	   284 MB/s
```

Perhaps, we should use split implementations to use REGS BLAKE2b and SSE2 BLAKE2s?

**Comment by mgorny** [comment]
Oh, I'm sorry. My statements were referring to BLAKE2b purely. Nevertheless, BLAKE2s is also 2 times slower for me ;-f.

**Comment by dchest** [comment]
Fixed in 1.0.0
